proto/buf/alpha/registry/v1alpha1/image.proto

```diff
@@ -25,10 +25,90 @@ service ImageService {
   rpc GetImage(GetImageRequest) returns (GetImageResponse);
 }
 
+// ImageMask is used in GetImageRequest to specify which parts of an image are
+// requested.
+enum ImageMask {
```

Comment:
This is now the proposal to replace the fieldmask, a specific enum over the repeateds we may want to include.

Alternatively, a `bool services_only` would be good enough for now, until we do the work for a better field mask. Both the this enum and a services only bool should be easy to convert in whatever new fieldmask we come up with, so I'm not strongly tied to either.